arm-relay/2016-07-01/relay.json

```diff
@@ -0,0 +1,2144 @@
+        },
+        "requiresClientAuthorization": {
+          "type": "boolean",
+          "description": "true if client authorization is needed for this relay; otherwise, false."
+        },
+        "authorizationRules": {
+          "$ref": "#/definitions/AuthorizationRules"
+        },
+        "requiresTransportSecurity": {
+          "type": "boolean",
+          "description": "true if transport security is needed for this relay; otherwise, false."
+        },
+        "isDynamic": {
+          "type": "boolean",
+          "description": "true if the relay is dynamic; otherwise, false."
+        },
+        "userMetadata": {
+          "type": "string",
+          "description": "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."
+        },
+        "collectionName": {
+          "type": "string",
+          "description": "The name of the collection associated with the relay."
+        }
+      },
+      "description": "Properties of the Namespace."
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of this Sku",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ],
+      "description": "Sku of the Namespace."
+    },
+    "AuthorizationRules": {
+      "type": "object",
+      "properties": {
+        "requiresEncryption": {
+          "type": "boolean",
+          "description": "true if authorization rule requires encryption; otherwise, false."
+        },
+        "count": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of authorization rule"
+        }
+      },
+      "description": "Authorization rule of the Namespace."
+    },
+    "SharedAccessAuthorizationRuleCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "data center location."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the AuthorizationRule."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SharedAccessAuthorizationRuleProperties"
+        }
```

Comment:
Is "properties" property here that references "SharedAccessAuthorizationRuleProperties" a required property here? If so,, please mark it as required